src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/CoreCLR/WebRequestPSCmdlet.CoreClr.cs

```diff
@@ -847,17 +847,20 @@ internal void ParseLinkHeader(HttpResponseMessage response, System.Uri requestUr
             IEnumerable<string> links;
             if (response.Headers.TryGetValues("Link", out links))
             {
-                foreach(string link in links.FirstOrDefault().Split(","))
+                foreach (string linkHeader in links)
                 {
-                    Match match = Regex.Match(link, pattern);
-                    if (match.Success)
+                    foreach (string link in linkHeader.Split(","))
                     {
-                        string url = match.Groups["url"].Value;
-                        string rel = match.Groups["rel"].Value;
-                        if (url != String.Empty && rel != String.Empty && !_relationLink.ContainsKey(rel))
+                        Match match = Regex.Match(link, pattern);
+                        if (match.Success)
                         {
-                            Uri absoluteUri = new Uri(requestUri, url);
-                            _relationLink.Add(rel, absoluteUri.AbsoluteUri.ToString());
+                            string url = match.Groups["url"].Value;
+                            string rel = match.Groups["rel"].Value;
+                            if (url != String.Empty && rel != String.Empty && !_relationLink.ContainsKey(rel))
```

Comment:
I don't know if `match.Success` is true enough to exclude an exception. It seems safer to check `String.IsNullOrEmpty()` for `url` and `rel`.

Reply:
> I don't know if match.Success

In this instance, it probably is. We are assured a string for both link and pattern so long as we don't mess with the pattern `Regex.Match` will never except and it true it the 2 indexes will exist in matches.

I agree on the `String.IsNullOrEmpty()` but the point of this PR is to address the Multiple Link Header issue, not to change this logic. Should that be done in a separate PR?

Reply:
[Doc](https://msdn.microsoft.com/en-us/library/hchbwda2(v=vs.110).aspx) says that there is no null.

Closed.